.github/workflows/java-gradle-release.yaml

```diff
@@ -79,10 +79,8 @@ jobs:
       - name: Bump version
         id: evaluate-version
         run: |
-          old_version=$(grep "version=" gradle.properties | cut -d "=" -f2)
-          release_version=$(semver bump "${{ inputs.release-type }}" "${old_version}")
-          echo ::set-output "name=old-version::$old_version"
-          echo ::set-output "name=release-version::$release_version"
+          echo "old-version=$(grep "version=" gradle.properties | cut -d "=" -f2)" >> $GITHUB_OUTPUT
+          echo "release-version=$(semver bump "${{ inputs.release-type }}" "${old_version}")" >> $GITHUB_OUTPUT
```

Comment:
@yannick-roeder I am not really sure about the double quotes here

Reply:
@yannick-roeder Should shellcheck control this?

Reply:
I think the spellcheck is fine in general. If it is that complicated to change it in this MR, then you can remove it here and we will fix it with [this MR](https://github.com/bakdata/ci-templates/pull/49) where we are rewriting the output anyway